src/index.js

```diff
@@ -360,6 +367,42 @@ const unexpectedEnzyme = {
         expect(actualClasses, '[not] to contain', ...className.split(' '));
       }
     );
+
+    childExpect.exportAssertion(
+      '<ReactElement> [not] to equal <ReactElement>',
+      (expect, a, b) => {
+        const equal = expect.equal(a, b);
+        const not = expect.flags.not;
+        if (not === equal) {
+          if (not) {
+            expect.fail();
+          }
+
+          const diffResult = htmllike.diff(adapter, a, b, expect);
+
+          return htmllike.withResult(diffResult, result => {
+            if (result.weight !== 0) {
+              return expect.fail({
+                diff: function(output, diff, inspect) {
+                  return {
+                    diff: htmllike.render(result, output, diff, inspect)
+                  };
+                }
+              });
+            } else {
+              expect.fail();
+            }
+          });
+        }
+      }
+    );
+
+    childExpect.exportAssertion(
+      '<ReactWrapper> [not] to equal <ReactElement>',
```

Comment:
I'm getting second thoughs on this name. Maybe we use `to exhaustively satisfy`.

Reply:
It would also make sense to satisfy against an array of react elements.